build.gradle

```diff
@@ -48,6 +49,35 @@ flyway {
     placeholderSuffix = ']'
 }
 
+sourceSets {
+    integrationTest {
+        java {
+            compileClasspath += main.output + test.output
+            runtimeClasspath += main.output + test.output
+            srcDir file('src/integration-test/java')
```

Comment:
After spending more time with the code, if we could move these to the more standard test folder I think that'd be best as IntelliJ IDEA doesn't really see them in the packages view when it's broken out like this.  See https://github.com/OpenLMIS/openlmis-template-service/pull/8#commitcomment-17795246